assets/dev/js/editor/container/container.js

```diff
@@ -209,6 +209,57 @@ export default class Container extends ArgsObject {
 		return result;
 	}
 
+	/**
+	 * Function getUtilizedControls().
+	 *
+	 * Should return all controls that effecting the container.
+	 */
+	getUtilizedControls() {
+		const result = {},
+			activeControls = this.settings.getActiveControls();
+
+		Object.entries( activeControls ).forEach( ( [ controlName, control ] ) => {
+			const controlValue = this.settings.get( control.name );
+
+			if ( control.global && ! controlValue?.length ) {
+				if ( this.globals.get( control.name )?.length || this.getGlobalDefault( controlName ).length ) {
+					control.global.utilized = true;
+
+					result[ controlName ] = control;
```

Comment:
Check with @StyleShit how to check if it's a global value.

Reply:
@StyleShit does its valid?

Reply:
I already said from the beginning, `isGlobalApplied` isn't reliable,
@matipojo

Reply:
How is the `isGlobalApplied` different from this check?